### PR TITLE
update pathy

### PR DESCRIPTION
**Pull request by safareli**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
   "dependencies": {
     "purescript-prelude": "^3.1.0",
     "purescript-matryoshka": "^0.3.0",
-    "purescript-pathy": "^4.0.0",
+    "purescript-pathy": "safareli/purescript-pathy#refactor",
```
Comment (safareli) [inline bower.json]: 💀

**Comment by garyb** [comment]
LGTM once we make the Pathy release 👍

File: src/SqlSquared/Signature/Relation.purs [hunk 2]
```diff
@@ -36,7 +36,7 @@ type VariRelR =
   }
 
 type TableRelR =
-  { path ∷ Pt.AnyFilePath
+  { path ∷ Pt.AnyFile
```
Comment (cryogenian) [inline src/SqlSquared/Signature/Relation.purs:39]: Would you mind make this `Either AnyFile AnyPath` or do you think this is matter of other pr? 
E.g. marklogic directory querying

Comment (safareli) [reply src/SqlSquared/Signature/Relation.purs:39]: I think other pr would be better for this :d

**Review by cryogenian — APPROVED** [review]

**Review by cryogenian — APPROVED** [review]

**Pull request merged**